ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-sno.yaml

```diff
@@ -21,7 +21,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-aws-control-plane-sno-periodic
+- as: perfscale-aws-control-plane-sno-periodic
```

Comment:
this test and the 4.13 sno test are named the exact same. did you add version to all test names?

Reply:
will the periodic names add the version?

Reply:
not sure I understand. we don't mention version number in the test names. may be I am not understanding the ask here

Reply:
don't know if its an issue that this test name and the sno test for 4.13 are the exact same name